spec/test2_spec.rb

```diff
@@ -0,0 +1,55 @@
+# spec/solution_spec.rb
+require_relative '../test2' # Assuming your solution is in a file named solution.rb
+
+RSpec.describe 'solution' do
+  class Tree
+    attr_reader :v, :l, :r
+
+    def initialize(v, l, r)
+      @v, @l, @r = v, l, r
+    end
+  end
```

Comment:
Still need it? It repeats to codebase on `test2.rb`. You may try by removing this class on test and run test suit again.